pkg/server/server.go

```diff
@@ -389,8 +391,23 @@ func (s *Server) HTTPHandler() http.Handler {
 	handle("/api/console/knative-channels", authHandler(s.handleKnativeChannelCRDs))
 	handle("/api/console/version", authHandler(s.versionHandler))
 
+	// Get running kubernetes version and inject into helm handlers.
+	client, err := discovery.NewDiscoveryClientForConfig(&rest.Config{
+		APIPath:   s.KubeAPIServerURL,
+		Transport: s.K8sClient.Transport,
+	})
+	if err != nil {
+		klog.Warningf("Failed to get kube discovery client %s", err.Error())
+	}
+
+	kubeVersion, err := client.ServerVersion()
+	if err != nil {
+		klog.Warningf("Failed to get server version %s", err.Error())
+	}
+
```

Comment:
we should wrap this up into a function and move it under `helm` package

Reply:
Done, added it under `actions` packages, let me know if that doesn't fit well.